user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Low_Code_Apps/Style_guide.md

```diff
@@ -0,0 +1,281 @@
+
+- Picking good colors to start from is often a challenge. One way to approach this is to pick some colors from a layout you find online, like this sample below:
+
+   |Sample UI | Color palette | Resulting app with same color palette|
+   |----------|---------------|-----------------------|
+   |![Sample UI](~/user-guide/images/LCA_Style_Guide_Sample_UI.webp)|![Color palette](~/user-guide/images/LCA_Style_Guide_Color_Palette.png)|![Sample UI result](~/user-guide/images/LCA_Style_Guide_Sample_UI_Result.png)|
+
+## Layout
+
+- Make sure components on a page/panel are aligned vertically and horizontally.
+
+   | 👍 Good | 👎 Bad |
+   |------|-----|
+   |![Good alignment](~/user-guide/images/LCA_Style_Guide_Alignment_Good.png)|![Bad alignment](~/user-guide/images/LCA_Style_Guide_Alignment_Bad.png)|
+
+- Use consistent spacing and a consistent margin on the left and right edge of each page in a low-code app. Avoid stretching components to the edge of the page with no spacing.
+
+- Make sure the available space is used, so there are no large empty areas on a page. At the same time , do not clutter the interface with too much information. In a low-code app, you can hide additional information on panels instead.
+
+- Try to limit the number of different font styles and font sizes.
+
+- Adjust the size of components to fit the information they contain. This means you should for example avoid very large ring/state components or very small tables and timelines.
+
+- You **must** ensure that your layout has been tested on different screen sizes before you publish a low-code app. Make sure it also works on a smaller screen like a laptop and not only on big monitors.
+
+## Navigation
+
+In order to have consistent navigation, we recommend applying the following order for navigation in a low-code app:
+
+1. Sidebar: Main navigation to the different pages defined in the app.
+
+   ![Navigation via sidebar](~/user-guide/images/LCA_Style_Guide_Navigation_Pages.png)
+
+   > [!NOTE]
+   > Use a side panel to show additional information for each item on a page.
+
+1. Header bar: Navigation within a page.
+
+   This can open up a hidden page, in which case it makes sense to change the title of each page so users know where they are.
+
+   ![Navigation via menubar](~/user-guide/images/LCA_Style_Guide_Navigation_Menu.gif)
+
+> [!IMPORTANT]
+> It is generally **not** advised to use buttons for navigation.
+
+## Panels
+
+- Panels in a low-code app should primarily be used to show additional information about an item selected on a page. This can for example be a DOM form or additional details/metadata that were not shown on the original page.
+
+- Panels can be shown on the left or the right of the screen or as a pop-up window. Try to always show a panel in the same location in an app, with the same width, etc. As a default, we recommend showing DOM forms in a pop-up window and showing additional information about a selected table row in a side panel on the right.
```

Comment:
Make sure to look out for leading capital letters for sentences and their punctuation marks.